packages/core/admin/ee/server/controllers/user.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     }
 
     const { body } = ctx.request;
-    const cleanData = { ...body, email: get(body, `email`, ``).toLowerCase() };
+    const cleanData = { ...body, email: _.get(body, `email`, ``).toLowerCase() };
```

Comment:
```suggestion
    const cleanData = { ...body, email: get('email', body)?.toLowerCase() ?? '' };
```

Reply:
yes but no :) I kept the code as copied from CE for consistency and will refacto the 2 at once later :)